src/appReview/slice.ts

```diff
@@ -0,0 +1,23 @@
+import { createSlice } from '@reduxjs/toolkit'
+
+export interface State {
+  lastInteractionTimestamp: number | null
```

Comment:
ah, now that there is only one variable i am wondering if we should have thrown this in the `app` reducer instead as something like "lastAppReviewPromptTimestamp"...in general i think small specific redux stores are fine, but i am struggling to think of any reasons why we'd add to this store in the future. "appReview" is almost too specific, so i'm not sure whether we should keep this. wdyt?

Reply:
Back when it was three variables a small redux store was great, but now that we just have one it really does seem like a bit much. 🤔

Reply:
@kathaypacific to move this into the app reducer.

Reply:
Move to the app reducer in eaa797d8a4cc460f2f98a0d8dc1439e11a01ad59! The version with the slice, 668caf73ecf3146f71694de3177dc11b3cf5c523, is also passing CI.